build/source/engine/time_utils.f90

```diff
@@ -98,6 +98,7 @@ subroutine extractTime(refdate,iyyy,im,id,ih,imin,dsec,ih_tz,imin_tz,dsec_tz,err
 
  ! FIELD 2: Advance to the ih:imin:dsec string
  istart=nsub+1
+ if(istart>n) return  ! no more time info
```

Comment:
What is the purpose of this? Seems like this should be a separate PR

Reply:
Hm, this seems to be vestigial. It has been removed.